src/utils/cellSetOperations.js

```diff
@@ -68,10 +64,14 @@ const complement = (listOfSets, properties) => {
     (acc, curr) => new Set([...acc, ...curr]),
   );
 
+  const filteredCellIds = hierarchy.filter((rootCluster) => rootCluster.key === 'louvain')[0]
+    .children
+    .map((child) => child.key);
```

Comment:
You are getting the `child.key` here, are they really the `filteredCellIds`?

Reply:
oh oops it should be renamed to clusters